src/cowboy_swagger.erl

```diff
@@ -42,6 +42,13 @@
       #{ type => binary()
        , properties => property_obj()
        }}.
+-type parameters_definition_array() ::
+  #{parameter_definition_name() =>
+      #{ type => binary()
+       , items => #{ type => binary()
+                   , properties => property_obj()
+                   }
+       }}.
 -export_type([parameter_definition_name/0, property_obj/0]).
```

Comment:
Should we also export the new type? I think we should.

Reply:
Fixed